src/jobs/release-lerna-packages.yml

```diff
@@ -77,10 +77,18 @@ steps:
         - ~/.cache/yarn
         - ~/.config/yarn
 
+  # - run npm:release:validate to see if a release is necessary - will set env var NEEDS_RELEASE
+  - when:
+      condition:
+        not: <<parameters.dryrun>>
+      steps:
+        check-if-releasable
   # - if environment looks like it is prepped to do change cases, and we're on "latest" tag
```

Comment:
does it actually check CTC envs?

Reply:
@mshanemc If "it" is `npm:release:validate` then it does the same checks as the release management release cmd, but without the npm tag/publish, etc. The check just captures if the current project should be released or not and persists that determination in the env var "NEEDS_RELEASE".